tests/plugins/inventory/netbox/2.3.5/expected.json

```diff
@@ -0,0 +1,55 @@
+{
+  "hosts": {
+    "1-Core": {
+      "name": "1-Core",
+      "nornir_host": {
+        "id": 1,
+        "url": "http://localhost:8080/api/ipam/ip-addresses/1/",
+        "family": 4,
+        "address": "10.0.1.1/32"
+      },
+      "serial": "",
+      "vendor": "Juniper",
+      "asset_tag": null,
+      "site": "sunnyvale-ca",
+      "role": "rt",
+      "model": "mx480",
+      "nornir_nos": null
+    },
+    "2-Distribution": {
+      "name": "2-Distribution",
+      "nornir_host": {
```

Comment:
@bdlamprecht  This doesn't seem correct, I guess you wanted to put here just the address, not the object itself, is that correct?

Reply:
I don't quite follow your question here.

Reply:
Ah, yes, I see what you're referring to.

This line...
https://github.com/nornir-automation/nornir/blob/8b56f970ab1e9fd478e109a760404109b7a2f266/nornir/plugins/inventory/netbox.py#L40

Should actually look like this:
```
                    temp["nornir_host"] = d["primary_ip"]["address"]
```

I saw that before, but must have missed it when committing the changes. :blush:

Reply:
Cool, will update :)